src/impl/channels.js

```diff
@@ -6,7 +6,28 @@ var dispatch = require("./dispatch");
 var MAX_DIRTY = 64;
 var MAX_QUEUE_SIZE = 1024;
 
-var CLOSED = null;
+var CLOSED = (function() {
+  var closedToken;
+  try {
+    closedToken = Symbol.for('@@csp/CLOSED');
+  } catch(e) {
+    closedToken = Object('@@csp/CLOSED');
+  }
+  return closedToken;
+})();
```

Comment:
Wouldn't it make more sense to write this as:
`var CLOSED = (function() {
  try {
    return Symbol.for('@@csp/CLOSED');
  } catch(e) {
    return Object('@@csp/CLOSED');
  }
})()`

Removes unnecessary variables.